person.rb

```diff
@@ -0,0 +1,26 @@
+class Person
+  attr_reader :id
+  attr_accessor :name, :age
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = rand(1...1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  def can_use_services?
+    isof_age? || "#{@parent_permission}, He has parental permission"
+  end
+
+  private
+
+  def isof_age?
+    @age > 18
```

Comment:
- [ ] the `is_of_age?` private method is required to return true if @age is ***greater or equal*** to 18 and false otherwise, here you are using only the `greater than (>)`, it should be `>=`, kindly change this. 😄

Reply:
linters insisting on of_age instead of is_of_age that's why I used isof_age instead

Reply:
@AmineHLub can you please show a screenshot of this error?

Reply:
thank you for replying back. here's a ss
![image](https://user-images.githubusercontent.com/83329154/146065429-89caefd7-6e28-4ca3-8c52-5b8a8a22fbfc.png)

Reply:
Okey, please use the suggested name `of_age?` 😄

Reply:
done :+1: